### PR TITLE
reports implemented on the page.

### PR DESCRIPTION
**Pull request by AMCernas**
# Description

We implemented the reports section to the projects, we are using the gem chartkick and groupdate to do this, so the reports can be seen as graphics by the users authenticated as admins.

## Screenshots
image

image

image

### CODE REVIEW

**Review by DiegoMoralesBravo — APPROVED** [review]

**Review by Reiyirak — APPROVED** [review]

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -50,6 +55,17 @@
...
+  background-color: #1cadfc;
+  color: black;
+}
+
+.checkin_header nav h1 {
+  color: white;
+}
+
+
+
```
Comment (YilKanda) [inline app/assets/stylesheets/application.css:68]: There are unnecessary spaces, which you can remove.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -66,6 +82,11 @@
   width: 100%;
 }
 
+html {
+  height: 100%;
+}
+
+
```
Comment (YilKanda) [inline app/assets/stylesheets/application.css:89]: There are unnecessary spaces, which you can remove.

File: app/views/reports/generate_report.html.erb [hunk 3]
```diff
@@ -1,22 +1,52 @@
...
+                <% daysNo.each do |d| %>
+                <%= simple_format "#{d.strftime('%B / %d / %Y ')} \n" %>
+                <% end %>
+            </td>
+            </tr>
+        <% end %>
+        </table>
+    </div>
+
 </div>
```
Comment (YilKanda) [inline app/views/reports/generate_report.html.erb:51]: Check the indentation of your file. (Space: 2)

File: config/importmap.rb [hunk 4]
```diff
@@ -5,3 +5,5 @@
 pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
 pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
 pin_all_from "app/javascript/controllers", under: "controllers"
+pin "chartkick", to: "chartkick.js"
+pin "Chart.bundle", to: "Chart.bundle.js"
```
Comment (YilKanda) [inline config/importmap.rb:9]: Add a blank line.

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by MarcoRodriguez23 — APPROVED** [review]
LGFM !!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**